### ISSUE TITLE
Request log shows "http" even when scheme is set to "https"

### ISSUE THREAD

**Issue by sectorsize512** [IBM/og#8]
To reproduce,

1) Add the following field to og's JSON config file:

```
"scheme":"https"
```

2) Run the benchmark with the config file

3) Check latest/request.log:

```
{"operation":"WRITE","type":"http","remote_address":"s3-api.sjc-us-geo.objectstorage.softlayer.net","user":"XXXX","timestamp_start":"1525900188512","timestamp_finish":"1525900189718","time_start":"09/May/2018:14:09:48 -0700","time_finish":"09/May/2018:14:09:49 -0700","request_method":"PUT","request_uri":"/newcont/XXX","object_id":"XXXX","status":200,"request_length":"1000000","user_agent":"og-1.7.0","request_latency":"1206","client_request_id":"6","request_id":"597c4b86-6617-4626-b4ee-dbafd6e6e170","stat":{"request_content":152.87,"close_latency":495.44,"ttfb":1204.96,"response_content":0.04,"total":1206.75},"object_length":"1000000","object_name":"XXX"}
```

The benchmark seem to run correctly, however.

**Comment by nlahmedIBM** [comment]
Thanks for raising the issue.  I will looking into it and put it in future release.

**Comment by sectorsize512** [comment]
thanks, @nlahmedIBM

**Comment by nlahmedIBM** [comment]
Fixed in Release 1.8.0
